osbs/build/build_request.py

```diff
@@ -228,7 +228,9 @@ def set_label(self, name, value):
         self.template['metadata']['labels'][name] = value
 
     def render_reactor_config(self):
-        if self.spec.reactor_config_secret.value is None:
+        if self.spec.reactor_config_secret.value is None and \
+                not self.spec.reactor_config_map.value and \
```

Comment:
It's a little confusing that for some values we're comparing to None while others verifying they're falsy. Can we make it the same? `is None` seems like a reasonable check for all.

Reply:
ok